lib/widget/dropdown.dart

```diff
@@ -210,14 +238,25 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     if (items != null) {
       results = [];
       for (SelectOneItem item in items) {
-        item.icon != null
-            ? results.add(
+        item.isIcon != null || item.isIcon == true
```

Comment:
"item.isIcon == true" requires no need for null check

Reply:
done